src/Fixer/Phpdoc/PhpdocAlignFixer.php

```diff
@@ -139,7 +147,7 @@ protected function createConfigurationDefinition()
             ->setAllowedValues([
                 $generator->allowedValueIsSubsetOf(self::$alignableTags),
             ])
-            // By default, all tags apart from @property will be aligned for backwards compatibility
+            // By default, all tags apart from @property and @method will be aligned for backwards compatibility
```

Comment:
Should I change this comment to be prefixed with `@TODO 3.0`?

Reply:
Yes please :)

Reply:
Done